src/datetime/mod.rs

```diff
@@ -409,8 +408,12 @@ impl<Tz: TimeZone> DateTime<Tz> {
     /// This does not overflow or underflow at all.
     #[inline]
     #[must_use]
-    pub fn signed_duration_since<Tz2: TimeZone>(self, rhs: DateTime<Tz2>) -> OldDuration {
-        self.datetime.signed_duration_since(rhs.datetime)
+    pub fn signed_duration_since<B, Tz2>(self, rhs: B) -> OldDuration
```

Comment:
Would prefer to spell this as `rhs: impl Borrow<DateTime<Tz2>>` -- the reduced indirection makes this easier to understand IMO. It might even be possible to replace the `Tz2` here with `impl TimeZone` as well. (Since we're not using it, do we even need the `Tz2: TimeZone` bound?)

Reply:
Nice, new syntax! Removing the `Tz2: TimeZone` didn't work.